packages/contentful/migrations/0100-create-landing-page.cjs

```diff
@@ -0,0 +1,434 @@
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  illustrationGroup
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        size: {
+          max: 6
+        }
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['illustration']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  // Create image card group to be used by landing page
+  const imageCardGroup = migration
+    .createContentType('imageCardGroup')
+    .name('Image card group')
+    .description('Section with text and grouped image cards')
+    .displayField('name');
+
+  imageCardGroup
+    .createField('name')
+    .name('Name')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  imageCardGroup
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  imageCardGroup
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        size: {
+          max: 6
+        }
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['imageCard']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  // Create sub section to be used by landing page
+  const subSection = migration
+    .createContentType('subSection')
+    .name('Sub section')
+    .description('Section with text and sections')
+    .displayField('name');
+
+  subSection
+    .createField('name')
+    .name('Name')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  subSection
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  subSection
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        size: {
+          max: 6
+        }
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['automatedCardGroup', 'illustrationGroup']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  // Create landing page
+  const landingPage = migration
+    .createContentType('landingPage')
+    .name('Landing page')
+    .description('Page with a hero and several content sections')
```

Comment:
I don't think we use 'hero' as a label anywhere else. 
`Page with a CTA header and additional info sections`

Reply:
Updated.